rohmu/object_storage/azure.py

```diff
@@ -67,13 +71,13 @@ class AzureTransfer(BaseTransfer[Config]):
 
     def __init__(
         self,
-        bucket_name,
-        account_name,
-        account_key=None,
-        sas_token=None,
-        prefix=None,
-        azure_cloud=None,
-        proxy_info=None,
+        bucket_name: str,
+        account_name: str,
+        account_key: Optional[str] = None,
+        sas_token: Optional[str] = None,
+        prefix: Optional[str] = None,
+        azure_cloud: Optional[str] = None,
+        proxy_info: Optional[dict[str, str]] = None,
```

Comment:
this is not entirely correct,
proxy_info has `port` which should be of type `int`

use `ProxyInfo` instead ?